fe/fe-core/src/test/java/com/starrocks/sql/plan/ReplayFromDumpTest.java

```diff
@@ -302,20 +302,21 @@ public void testTPCDS78() throws Exception {
         // check outer join with isNull predicate on inner table
         // The estimate cardinality of join should not be 0.
         Pair<QueryDumpInfo, String> replayPair = getCostPlanFragment(getDumpInfoFromFile("query_dump/tpcds78"));
+        System.out.println(replayPair.second);
         Assert.assertTrue(replayPair.second.contains("3:HASH JOIN\n" +
                 "  |  join op: LEFT OUTER JOIN (BUCKET_SHUFFLE)\n" +
                 "  |  equal join conjunct: [257: ss_ticket_number, INT, false] = [280: sr_ticket_number, INT, true]\n" +
                 "  |  equal join conjunct: [256: ss_item_sk, INT, false] = [279: sr_item_sk, INT, true]\n" +
                 "  |  other predicates: 280: sr_ticket_number IS NULL\n" +
                 "  |  output columns: 256, 258, 260, 266, 267, 269\n" +
-                "  |  cardinality: 39142590"));
+                "  |  cardinality: 37372757"));
```

Comment:
why the cardinality will change?

Reply:
A new predicate is derived, the old process has led to optimization not being executed